dev/docs/assets/overrides.css

```diff
@@ -125,11 +111,6 @@ i.md-36 {
   vertical-align: bottom;
 }
 
-/* thinify the inherited names in lists */
-li.inherited a {
-  font-weight: 100;
```

Comment:
100 might be too light, be they should be differentiated.

Reply:
Maybe 300 here?

Reply:
The dartdoc default stylesheet already gives these a [300 weight](https://github.com/dart-lang/dartdoc/blob/ff0d94bdb87f11c04a7e0ddab811bf94211b08a4/lib/resources/styles.css#L1001-L1012). There doesn't seem a need to additionally put something in the override.

See [api.dart.dev](https://api.dart.dev/stable/2.15.0/dart-core/bool-class.html) for an example of how it would look.

Reply:
Ahh, OK, if they're already 300, then sure.